dockers/docker-dhcp-relay/isc-dhcp-relay.j2

```diff
@@ -0,0 +1,7 @@
+SERVERS="{{ dhcp_servers | join(' ') }}"
+
+INTERFACES="{{ minigraph_vlan_interfaces | join(' ') }}"
```

Comment:
This template will only be used to generate the config file if the docker is deployed and started, which will only occur if the device is a ToR. The actual decision on whether or not to deploy the docker will be controlled by the mgmt repo. I felt that logic was best left out of the docker itself.